scripts/postinstall.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 mkdir libs
-cp node_modules/wikipedia-preview/dist/wikipedia-preview.production.js libs/
+cp node_modules/wikipedia-preview/dist/wikipedia-preview.js libs/
```

Comment:
So wikipedia-preview.js instead of wikipedia-preview.umd.cjs? I think testing will help me understand this better